crates/sui-framework/packages/sui-framework/sources/crypto/nitro_attestation.move

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::nitro_attestation;
+
+use sui::clock::{Self, Clock};
+
+#[allow(unused_const)]
+/// Error that the feature is not available on this network.
+const ENotSupportedError: u64 = 0;
+#[allow(unused_const)]
+/// Error that the attestation input failed to be parsed. 
+const EParseError: u64 = 1;
+#[allow(unused_const)]
+/// Error that the attestation failed to be verified. 
+const EVerifyError: u64 = 2;
+#[allow(unused_const)]
+/// Error that the pcrs length is invalid. 
+const EInvalidPcrLength: u64 = 3;
+
+/// Represents a PCR entry with an index and value.
+public struct PCREntry has drop {
+    index: u8,
+    value: vector<u8>
+}
+
+/// Nitro Attestation Document defined for AWS.
+public struct NitroAttestationDocument has drop  {
+    /// Version
+    version: u8,
```

Comment:
what's the purpose of version?

Reply:
you left a comment before to add this version? should I remove it?

Reply:
Which comment was it? maybe it was regarding backward compatibility of the pcr ordering, and it might not be needed no